app/components/atoms/button/button.stories.ts

```diff
@@ -0,0 +1,18 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import Button from './button';
+
+const meta: Meta<typeof Button> = {
+    title: 'Atoms/Button',
+    component: Button
+  };
+
+  export default meta;
+  
+  type Story = StoryObj<typeof Button>;
+  
```

Comment:
Make sure all the props for the button shows up in the storybook docs. Currently `type` doesn't render. I believe you need to add it in the `args`.